meshroom/core/graph.py

```diff
@@ -1564,6 +1566,10 @@ def cacheDir(self, value):
         self.updateStatusFromCache(force=True)
         self.cacheDirChanged.emit()
 
+    @property
+    def viewpoint(self):
```

Comment:
selectedViewpoint/activeViewpoint/currentViewpoint?

Reply:
a docstring to explain too

Reply:
Done